include/LayerPlan.h

```diff
@@ -398,11 +401,15 @@ class LayerPlan : public NoCopy
         const Polygon& polygon,
         int startIdx,
         const bool reverse,
+        const Settings& settings,
         const GCodePathConfig& config,
         coord_t wall_0_wipe_dist = 0,
         bool spiralize = false,
         const Ratio& flow_ratio = 1.0_r,
-        bool always_retract = false);
+        bool always_retract = false,
+        bool scarf_seam = false,
+        bool smooth_speed = false,
+        bool is_candidate_small_feature = false);
```

Comment:
I'm pretty sure I'm overlooking something, but where is this filled? Is it always `false`? I know it's used in the check for 'small features' below somewhere.

Reply:
Oh you're right, in the end it is never actually used, I'll remove it